eks-worker-al2.json

```diff
@@ -202,13 +204,19 @@
       "type": "shell",
       "remote_folder": "{{ user `remote_folder`}}",
       "script": "{{template_dir}}/scripts/generate-version-info.sh",
-      "execute_command": "chmod +x {{ .Path }}; {{ .Path }} /tmp/version-info.json"
+      "execute_command": "chmod +x {{ .Path }}; {{ .Path }} {{user `working_dir`}}/version-info.json"
```

Comment:
This will change. Is that ok? When we upload this after we build, it will break that, right?

Reply:
Nope, this is the location of the file on the builder instance. We download it to the local PWD after build and go from there.